sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -200,24 +200,36 @@ await receiver.CloseAsync();
 ```
 
 In v7:
-
-```C# Snippet:ServiceBusSendAndReceiveBatch
+```C# Snippet:ServiceBusInitializeSend
 string connectionString = "<connection_string>";
 string queueName = "<queue_name>";
 // since ServiceBusClient implements IAsyncDisposable we create it with "await using"
 await using var client = new ServiceBusClient(connectionString);
 
 // create the sender
 ServiceBusSender sender = client.CreateSender(queueName);
+```
+```C# Snippet:ServiceBusSendAndReceiveBatch
+IList<ServiceBusMessage> messages = new List<ServiceBusMessage>();
+messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("First")));
+messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("Second")));
+// send the messages
+await sender.SendAsync(messages);
```

Comment:
Can we update the description [here](https://github.com/JoshLove-msft/azure-sdk-for-net/blob/preview2-readme/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#sending-and-receiving-a-batch-of-messages) and talk about SendAsync(IEnumerable<ServiceBusMessage>) api?